backend/game/engine/player_queue.py

```diff
@@ -36,7 +36,10 @@ def reset_allowed_to_yield(self):
 
     def get_next_player(self):
         self.reset_allowed_to_yield()
-        self.current_player = self._cycle_next_alive_player()
+        if self.current_player is not None and self.current_player.gets_bonus_turn:
```

Comment:
this can be reduced. Unless can think of a specific reason for it "is not None" is often not needed as we can just check for it using a regular if

`if self.current_player and self.current_player.gets_bonus_turn:`

Reply:
Yeah that makes sense. Thanks